crypto/zkp/primitives/commitments/damgard-fujisaki_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2017 XLAB d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package commitmentzkp
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/xlab-si/emmy/crypto/commitments"
+	"github.com/xlab-si/emmy/crypto/common"
+	"math/big"
+	"testing"
+)
+
+// TestProveDFCommitmentOpening demonstrates how to prove that you can open DamgardFujisaki commitment.
+func TestProveDFCommitmentOpening(t *testing.T) {
```

Comment:
I suggest renaming this test to something that includes `DamgardFujisaki`. For instance, `TestOpenDamgardFujisakiCommitment` (or whatever you find most appropriate). It produces a clearer output when the tests are actually run.